### PR TITLE
Update sample method docstrings

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Some minor updates to the `sample()` methods from user feedback.

### CODE REVIEW

File: dwave/samplers/planar/sampler.py [hunk 1]
```diff
@@ -39,11 +39,11 @@ def sample(self,
...
                pos: typing.Optional[typing.Mapping[dimod.typing.Variable, typing.Tuple[float, float]]] = None,
                **kwargs) -> dimod.SampleSet:
-        """Sample from a binary quadratic model.
+        """Find a ground state of the problem.
 
         Args:
-            bqm: Binary quadratic model to be sampled.
-            pos: Position for each node
+            bqm: Binary quadratic model representing a planar Ising model with no linear biases.
+            pos: Position for each node.
```
Comment (arcondello) [inline dwave/samplers/planar/sampler.py:46]: While we're editing this. We can probably say that the position is given by a dict of the form `{node: (x-coordinate, y-coordinate),...}`

File: dwave/samplers/planar/sampler.py [hunk 2]
```diff
@@ -39,11 +39,11 @@ def sample(self,
                bqm: dimod.BinaryQuadraticModel,
                pos: typing.Optional[typing.Mapping[dimod.typing.Variable, typing.Tuple[float, float]]] = None,
                **kwargs) -> dimod.SampleSet:
-        """Sample from a binary quadratic model.
+        """Find a ground state of the problem.
```
[the remark this answers is not in the document]
Comment (JoelPasvolsky) [reply dwave/samplers/planar/sampler.py:42]: I considered that and my reasoning here is that in this case the full description is short, "planar Ising model with no linear biases", but that's not always the case, so you end up with an inconsistency on whether or not the method describes the problem/model. Given that the problem is described just above in the class and just below in the parameter, I don't think another description here adds too much needed info

File: dwave/samplers/planar/sampler.py (hunk 1 above)
Comment (JoelPasvolsky) [reply dwave/samplers/planar/sampler.py:46]: good idea

File: dwave/samplers/planar/sampler.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (arcondello) [reply dwave/samplers/planar/sampler.py:42]: If we can fit it on the first line, I think we should. For all of those users who just read [the class docs](https://docs.ocean.dwavesys.com/en/stable/docs_samplers/reference.html#methods).

**Review by arcondello — APPROVED** [review]

**Pull request merged**